cost_function.py

```diff
@@ -0,0 +1,20 @@
+import math
+from hypothesis_function import HypothesisFunction
+
+class CostFunction:
+    def __init__(self, training_set):
```

Comment:
@lucyfox4131 That's really funny! I think the underscore syntax is a little strange, but I guess it takes some getting used to. Overall, Python looks pretty clean